downstream/titles/aap-operator-installation/master.html

```diff
@@ -4,7 +4,7 @@
 <meta charset="UTF-8">
```

Comment:
@dcdacosta Hi, I know this PR is currently a WIP but I noticed the `master.html` file is included here. There is a `.gitignore` file in the repo that should keep it from getting picked up. I'm not sure why that isn't working but you might want to delete this so it doesn't get added to the repo. Cheers.

Reply:
Done!